README.md

```diff
@@ -55,6 +55,7 @@ Group | Description
 com.decathlon.android.foundation | Fundamental components of UI with texts, icons and colors.
 com.decathlon.android.buttons | Build UIs with ready to use Button components.
 com.decathlon.android.checkboxes | Build UIs with ready to use Checkbox components.
+com.decathlon.android.radio | Build UIs with ready to use RadioButton components.
```

Comment:
Should be written with an `s` to be aligned with other components modules ? Or even `radiobuttons` ?

Reply:
Agree, I was expecting a feedback on this decision. `radio` (singular) or `radiobuttons` (plural) are fine to me. I personally decided to go for `radio` because it was more pleasant to read but maybe a bit more obscur.